docs/advanced-topics/snapshot-and-doi.rst

```diff
@@ -0,0 +1,101 @@
+Creating Snapshots and Requesting DOIs
+=============================================
+
+Introduction
+------------------
+Starting in Dockstore 1.7.0, users can create snapshots and Digital
```

Comment:
@agduncan94 Is specifying Dockstore version still necessary? I thought readthedocs is supposed to be a solution for this problem

Reply:
Sphinx has things like the following https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded

Can also use the note section. Since we are releasing the docs and 1.7.0 at the same time though I don't know if it is even necessary to include.